src/api/v1/paths/onbase/documents.js

```diff
@@ -0,0 +1,23 @@
+import { errorBuilder, errorHandler } from 'errors/errors';
+
+import { postDocument } from '../../db/oracledb/onbase-dao';
```

Comment:
I could be wrong but I think we can just do `db/oracledb/onbase-dao`

Reply:
This API is still using swagger 2 version skeleton, so we don't have root deirectory alias set up for this path resolver.